x-pack/plugins/lens/public/app_plugin/shared/edit_on_the_fly/lens_configuration_flyout.tsx

```diff
@@ -477,8 +480,8 @@ export function LensEditConfigurationFlyout({
             `}
                 >
                   <h5>
-                    {i18n.translate('xpack.lens.config.layerConfigurationLabel', {
-                      defaultMessage: 'Layer configuration',
+                    {i18n.translate('xpack.lens.config.chartConfigurationLabel', {
+                      defaultMessage: 'Chart configuration',
```

Comment:
The Add layer appears if you try to edit a dataview mode Lens chart (not ESQL). We are currently supporting only one layer in ESQL charts.

About the naming I am personally comfortable with whatever you think is better Michael. 

So to wrap up:

- I have always one layer in ESQL charts
- I can have multiple layers in dataview XY charts

Which is the correct label to describe best all our cases? Visualization or Layer? I am fine with whatever you prefer!

Reply:
I prefer to stick to Visualization configuration because it encapsulates many layers settings. 

About your question
> The Add layer appears if you try to edit a dataview mode Lens chart (not ESQL). We are currently supporting only one layer in ESQL charts. 
- this one is a bug, it shouldn't appear yet. I've hidden it and will work on it in the second phase.